web/src/components/CopyableTextField/index.tsx

```diff
@@ -56,31 +57,18 @@ class CopyableTextField extends React.Component<CopyableTextFieldProps> {
         const cx = classnames('cr-copyable-text-field', className);
 
         return (
-            <div className={cx}>
-                <InputGroup>
-                    <CustomInput
-                        id={String(fieldId)}
-                        readOnly={true}
-                        inputValue={value}
-                        handleClick={doCopy}
-                        type="text"
-                        isDisabled={disabled}
-                        label={label || ''}
-                        defaultLabel={label || ''}
-                        placeholder={label || ''}
-                    />
-                    <InputGroup.Append>
-                        <Button
-                            onClick={doCopy}
-                            disabled={disabled}
-                            size="lg"
-                            variant="primary"
-                        >
-                            {copyButtonText ? copyButtonText : 'Copy'}
-                        </Button>
-                    </InputGroup.Append>
-                </InputGroup>
-            </div>
+          <InputWithButton
+            value={value}
+            className={cx}
+            fieldId={fieldId}
+            readOnly
+            handleClickInput={doCopy}
+            handleClickButton={doCopy}
+            type="text"
+            disabled={disabled}
+            label={label}
+            buttonText={copyButtonText}
```

Comment:
```suggestion
            buttonText={copyButtonText || 'Copy'}
```

Reply:
this will be "" (empty string) by default